ubiconfig/_impl/loaders/local.py

```diff
@@ -38,7 +43,9 @@ def load_all(self, recursive=False):
         file_list = self._get_local_file_list(recursive)
         for file in file_list:
             LOG.debug("Now loading %s", file)
-            ubi_configs.append(self.load(file))
+            config = self.load(file)
```

Comment:
And what about yaml.YAMLError? I think would be better to try here for both exceptions.

Reply:
If the syntax in the file is wrong it will raise YAMLError here, it should be added a try-except here I guess unless you want to break execution if the syntax is wrong, but I guess is the same as wrong schema.

Reply:
This is related to the raising instead of returning comment. We are handling YAMLError to log and raise, but the behavior for ValidationError is log and return. I think is best to log and raise for both, and add a try except here.

Reply:
> I think YAMLError and ValidationError should be treated equally.

Yes, you're right. Thank you for pointing this out.

Reply:
the YAMLError is caught in load and re-raised. I should log it and skip it there.